packages/iobio-react-components/src/utils/webComponentTools.mts

```diff
@@ -50,3 +52,48 @@ export const setElementStyles = (element: Element, styles: string) => {
 		element.shadowRoot.adoptedStyleSheets = [elementStyles];
 	}
 };
+
+/** File Strategy & Bed URL */
+const fileStrategies = ['WGS', 'WXS', 'ChipSeq', 'RNA-Seq'];
+type FileStrategyKey = (typeof fileStrategies)[number];
+export type DefaultBedUrls = {
+	[K in FileStrategyKey]: string;
+};
+
+// TODO: Finalize URLs after merge, these are just for testing
+const bedShuffled1BrowserUrl =
+	'https://raw.githubusercontent.com/overture-stack/iobio-components/29a2f8ec57ea8d38a00e998c52487b5aafe5095d/packages/iobio-react-components/src/utils/bedFiles/1k_flank_hg38_shuffled1.bed';
```

Comment:
See the todo, I will need to make a fast-follow up PR to update these URLs once these files are in `main`
These current URLs are based off the Draft PR.

Reply:
Also the comment here has been improved, Git UI is not showing it for me in this conversation